### ISSUE TITLE
RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.

### ISSUE THREAD

**Issue by angvp** [davidfischer/rpc4django#40]
Hi I'm getting this warning which might be far from happen now but may be in some days this must be fixed.

```
...
/lib/python2.7/site-packages/rpc4django/views.py:21: RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.
  from django.utils.importlib import import_module
```

I'm working in a pull request for this issue

**Comment by davidfischer** [comment]
I'm glad you're working on a PR. I'm not a user of this library I wrote anymore and so I don't maintain it extremely actively.

**Comment by angvp** [comment]
Oh I found this the only library who worked to me on a project where I have to simulate an old xml-rpc api if you're needing maintainers I could give you a hand.

**Comment by davidfischer** [comment]
Any help you can give would be amazing. I merged #42.
